app/models/service_template_filter.rb

```diff
@@ -0,0 +1,29 @@
+module ServiceTemplateFilter
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def include_service_template?(parent_svc_task, service_template_id, parent_svc = nil)
+      attrs = {'request' => 'SERVICE_PROVISION_INFO', 'message' => 'include_service'}
+      st = ServiceTemplate.find(service_template_id)
+      set_automation_attrs([parent_svc_task.get_user, st, parent_svc, parent_svc_task], attrs)
+      uri = MiqAeEngine.create_automation_object("REQUEST", attrs, :vmdb_object => parent_svc_task)
+      automate_result(uri, st.name)
+    end
+
+    def automate_result(uri, name)
```

Comment:
The method name `automate_result` is generic but the method is checking a very specific automate return value.  The name should be more descriptive like `automate_result_include_service_template?`.  Also, this method is not guaranteed to return a boolean value which I think is what you want and would fix the return value from `include_service_template?`.